script/cibuild

```diff
@@ -26,7 +26,7 @@ then
     fi
     docker run --env CC=$CC --env CXX=$CXX --env FLAGS=$FLAGS --name $DOCKER_CONTAINER_NAME $DST_DOCKER_IMG_NAME $FLAGS
```

Comment:
I think I finally figured out my problem. The IS_RELEASE and TAG_NAME environment variables simply weren't getting passed through to the Docker container. 🤦🏽‍♂️ 

To fix this, change this line to:

```bash
docker run --env CC=$CC --env CXX=$CXX --env IS_RELEASE=$IS_RELEASE --env TAG_NAME=$TAG_NAME --env FLAGS=$FLAGS --name $DOCKER_CONTAINER_NAME $DST_DOCKER_IMG_NAME $FLAGS
```

Reply:
updated